.github/CODEOWNERS

```diff
@@ -7,6 +7,9 @@
 # For documentation on this file, see https://help.github.com/articles/about-codeowners/
 # Mentioned users will get code review requests.
 
+# Release managers, release branch only
+/ @jonringer
```

Comment:
From the [example in the docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) seems this should be `*` instead of `/`.

> Ensure that all backport PRs are getting merged.

Last match takes precedence so you still won't get a notification if a PR only touches something that is listed below this.

Reply:
I think this still covers 99% of PRs. And there's not many which backport changes to the dot directories.

Good to know for future releases though :)

Reply:
I don't think `/` is matching anything?

Reply:
hard to know, I don't think there's been a backport to release-21.11 since the merge of this.

https://github.com/NixOS/nixpkgs/pull/158493 doesn't really count because I'm a python codeowner

Reply:
#158523

Reply:
damn :(. Not in my notificaitons

Reply:
Pushed.